templates/blocks/google_tag_manager.twig

```diff
@@ -57,6 +104,20 @@
       gtm_allow = false;
     }
 
+    {#
+      If Google Consent Mode is enabled,
+      set default ad storage and analytics storage to 'denied' as first action on every page until consent is given.
+      If consent given, update consent on every page.
+    #}
+    if ( p4.cookies.enable_google_consent_mode ) {
```

Comment:
We can do this condition in the Twig template already, and not output this part in JS. As a result we wouldn't need to define `enable_google_consent_mode` in JS anymore.